packages/collection-diff-apply/index.js

```diff
@@ -80,6 +80,9 @@ function diffApply(obj, diff, pathConverter) {
     }
     var thisProp;
     while (((thisProp = pathCopy.shift())) != null) {
+      if (thisProp === '__proto__' || thisProp === 'constructor' || thisProp === 'prototype') {
+        throw new Error('Prototype pollution attempt detected');
```

Comment:
No need to throw an exception here, we can just silently ignore with `break`. This way legitimate props still get applied.